src/__tests__/Calculator.test.js

```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
```

Comment:
- [ ] Kindly add tests that simulate user interaction as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Use%20React%20Testing%20Library%20to%20simulate%20user%20interaction.).

Reply:
It's done, we add a test simulating the pressed key "8" and then the key "7" and find the value "87" on the screen.